common/views/components/Header/HeaderPrototype.tsx

```diff
@@ -12,6 +12,22 @@ import {
   withUserInfo,
 } from '@weco/identity/src/frontend/MyAccount/UserInfoContext';
 
+const FreeStickerPortal = styled.div.attrs<{ isHidden: boolean }>(props => ({
+  id: 'free-sticker-portal',
+  className: classNames({
+    'is-hidden-s': props.isHidden,
+  }),
+}))<{ isHidden: boolean }>`
+  position: absolute;
+  right: ${props => props.theme.grid.s.gutter}px;
+  top: 100%;
+  z-index: 83;
```

Comment:
I don't know why this has to be `83` and below `84`, but they are the lowest numbers that make a difference. For the record, I'm not particularly happy about this.

Reply:
Is it worth setting up some z-index spaces abnd sticking them into vasriables? Not quite sure how this might look. Maybe something like:
```TS
theme.zIndex.header.component1 = 81
theme.zIndex.header.component2 = 82
theme.zIndex.header.component3 = 83

theme.zIndex.otherSpace.component1 = 91
```

That way we can see them all together?